### PR TITLE
EZP-28754: As a v1 Editor, I want to select a SiteAccess in the Preview mode

### PR DESCRIPTION
**Pull request by sunpietro**
https://jira.ez.no/browse/EZP-28754

zrzut ekranu 2018-01-19 o 14 17 38

### CODE REVIEW

**Comment by webhdx** [comment]
22z409

File: Resources/public/js/helpers/ez-eqeqeqhelper.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-eqeqeqhelper', function (Y) {
```
Comment (Nattfarinn) [inline Resources/public/js/helpers/ez-eqeqeqhelper.js]: `eqeqeqHelper`

1p54g9

**Comment by andrerom** [comment]
Ok, can we send this to QA?

**Comment by sunpietro** [comment]
@andrerom - rebased just now :)

**Comment by andrerom** [comment]
Thanks, if you rebase another time behat should also pass now ;)

**Comment by sunpietro** [comment]
done :)

**Comment by lserwatka** [comment]
@micszo we need a round of QA here

**Pull request merged**